frontend/src/metabase-lib/lib/metadata/Schema.ts

```diff
@@ -9,10 +7,10 @@ import Table from "./Table";
  */
 
 export default class Schema extends Base {
-  id?: string;
```

Comment:
I think `id?` is wrong here, but that's just a guess. When I looked in the app, I never saw any schemas without it.